editoast/editoast_models/src/rolling_stock/rolling_stock_category.rs

```diff
@@ -0,0 +1,66 @@
+use std::io::Write;
+use std::str::FromStr;
+
+use diesel::deserialize::FromSql;
+use diesel::deserialize::FromSqlRow;
+use diesel::expression::AsExpression;
+use diesel::pg::Pg;
+use diesel::pg::PgValue;
+use diesel::serialize::Output;
+use diesel::serialize::ToSql;
+use serde::Deserialize;
+use serde::Serialize;
+use strum::Display;
+use strum::EnumString;
+use strum::IntoStaticStr;
+use utoipa::ToSchema;
+
+editoast_common::schemas! {
+    RollingStockCategory,
+}
+
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    ToSchema,
+    FromSqlRow,
+    AsExpression,
+    EnumString,
+    IntoStaticStr,
+    Display,
+)]
+#[diesel(sql_type = crate::tables::sql_types::RollingStockCategory)]
+#[strum(serialize_all = "SCREAMING_SNAKE_CASE")]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum RollingStockCategory {
```

Comment:
Could you add a comment mentioning that this enum maps a Postgres enum and that any change made to this struct must be reflected in the DB and vice versa?